index.html

```diff
@@ -552,39 +563,150 @@ <h3>BitstringStatusListEntry</h3>
+                </p>
+              </td>
+            </tr>
           </tbody>
         </table>
 
-        <pre class="example nohighlight" title="Example StatusListCredential">
+
+        <p>
+Status list entries can be used to associate a single status type,
+such as `revocation` or `suspension`, with a [=verifiable credential=] by using
+the `statusPurpose` property. The example below demonstrates the association
+of simple status list entries:
+        </p>
+
+        <pre class="example nohighlight"
+             title="Example StatusListCredential using simple entries">
 {
   "@context": [
-    "https://www.w3.org/ns/credentials/v2"
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
   ],
   "id": "https://example.com/credentials/23894672394",
-  "type": ["VerifiableCredential"],
+  "type": ["VerifiableCredential", "EmployeeIdCredential"],
   "issuer": "did:example:12345",
-  "validFrom": "2021-04-05T14:27:42Z",
-  <span class="highlight">"credentialStatus": [
-    {
-      "id": "https://example.com/credentials/status/3#94567",
-      "type": "BitstringStatusListEntry",
-      "statusPurpose": "revocation",
-      "statusListIndex": "94567",
-      "statusListCredential": "https://example.com/credentials/status/3"
-    }, {
-      "id": "https://example.com/credentials/status/4#23452",
-      "type": "BitstringStatusListEntry",
-      "statusPurpose": "suspension",
-      "statusListIndex": "23452",
-      "statusListCredential": "https://example.com/credentials/status/4"
-    }
-  ]</span>,
+  "validFrom": "2024-04-05T14:27:42Z",
+  <span class="highlight">"credentialStatus": [{
+    "id": "https://example.com/credentials/status/3#94567",
+    "type": "BitstringStatusListEntry",
+    "statusPurpose": "revocation",
+    "statusListIndex": "94567",
+    "statusListCredential": "https://example.com/credentials/status/3"
+  }, {
+    "id": "https://example.com/credentials/status/4#23452",
+    "type": "BitstringStatusListEntry",
+    "statusPurpose": "suspension",
+    "statusListIndex": "23452",
+    "statusListCredential": "https://example.com/credentials/status/4"
+  }]</span>,
   "credentialSubject": {
     "id": "did:example:6789",
-    "type": "Person"
+    "type": "Person",
+    "employeeId": "A-123456"
   }
 }
         </pre>
+
+        <p>
+Status list entries can be used to associate many status types with a
+[=verifiable credential=] by using the `message` status purpose. The set of
+messages associated with a particular entry is committed to by the [=issuer=]
+by using the `statusSize`, `statusMessage`, and optional `statusReference`
+properties. This commitment is done at the time of issuance to ensure that
+the [=holder=] knows what sort of information might be associated with a
+particular [=verifiable credential=] that is in their possession, and would
+then be discoverable by a [=verifier=] that receives that credential.
```

Comment:
My attempt to incorporate everything above:

```suggestion
The use of `message` as the status purpose enables an issuer to
define an arbitrary number of custom, descriptive messages about
the status of the [=verifiable credential=]. The [=issuer=] commits to
the set of messages that may be associated with a particular entry
(i.e., with a particular [=verifiable credential=]) through the
`statusSize`, `statusMessage`, and optional `statusReference` properties,
at the time of [=verifiable credential=] issuance. This is to ensure that
the [=holder=] knows what sort of information might be associated with a
particular [=verifiable credential=] they keep in their possession, that
could then be discoverable by a [=verifier=] that later receives that
credential.
```